packages/admin-panel/src/layout/navigation/UserProfileInfo.jsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { Divider, Typography } from '@material-ui/core';
+import { UserLink } from './UserLink';
+
+const Wrapper = styled.div`
+  padding-inline: 1.25rem;
+  display: flex;
+  flex-direction: column;
+`;
+
+const UserName = styled(Typography)`
+  font-weight: ${props => props.theme.typography.fontWeightMedium};
+  font-size: 0.875rem;
+`;
+
+const UserEmail = styled(Typography)`
+  font-size: 0.6875rem;
+  margin-block-start: 0.25rem;
+  margin-block-end: 0.9rem;
```

Comment:
Wouldn’t normally comment on this, but on the off chance you don’t know the shorthand:

```suggestion
  margin-block: 0.25rem 0.9rem;
```